src/config.rs

```diff
@@ -363,8 +363,12 @@ impl Config {
         matches
             .get_one::<String>("release")
```

Comment:
this is trying to read `SENTRY_RELEASE` only after if tails to find a `release` in the clap match object right?